Added Non-Functional Login Screen

ErickHeyl opened the pull request:
Doesn’t do anything yet, but try to see if you can get it to run on
your device

ErickHeyl committed:
Added Non-Functional Login Screen

Doesn’t do anything yet, but try to see if you can get it to run on
your device

haleyminshell reviewed (approved):
I pulled down the branch and was able to open the project in QT creator. It works and I see our nonfunctional login page there.

allanhaywood reviewed (changes requested):
Change untitled to PlotTalk

ErickHeyl committed:
Changed Title to Plot Talk

removed the untitled project and created a new one with the correct
name for our project
